notebooks/WFC3/uvis_pam_corrections/WFC3_UVIS_Pixel_Area_Map_Corrections_for_Subarrays.ipynb

```diff
@@ -471,7 +471,7 @@
     "diff_data_512 = (pamcorr_data_512 - data_512)\n",
     "\n",
     "# We use zcale to find the min and max for plotting\n",
-    "vmin_diff_512, vmax_diff_512 = zscale(diff_data_512)\n",
+    "vmin_diff_512, vmax_diff_512 = z.get_limits(diff_data_512)\n",
```

Comment:
Are you importing `ZScaleInterval` as `z`?

Reply:
> Are you importing `ZScaleInterval` as `z`?

`ZScaleInterval` in the above code cells is initialized as z. I followed the convention in other notebooks.